src/components/data/_tests/data-utils.test.js

```diff
@@ -0,0 +1,63 @@
+import { concatenateAttributeNames } from 'src/components/data/data-utils.js'
+
+
```

Comment:
If you had, for example:
```suggestion
const mkEmpty = () => []
```